cypress-E2E-tests/cypress/support/device.commands.ts

```diff
@@ -80,158 +88,183 @@ declare global {
+    request.body['creationTime'] = currentDate.toISOString();
   }
   cy.apiRequest(request);
 });
 
-Cypress.Commands.add("deleteAllDevices", () => {
+Cypress.Commands.add('deleteAllDevices', () => {
   cy.apiRequest({
-    url: "inventory/managedObjects?fragmentType=c8y_IsDevice",
-    method: "GET",
-    failOnStatusCode: false,
+    url: 'inventory/managedObjects?fragmentType=c8y_IsDevice',
+    method: 'GET',
+    failOnStatusCode: false
   }).then((response: any) => {
     for (const mo of response.body.managedObjects) {
-      cy.log("Deleting ", mo.id);
+      cy.log('Deleting ', mo.id);
       cy.apiRequest({
-        method: "DELETE",
-        url: `/inventory/managedObjects/${mo.id}`,
+        method: 'DELETE',
+        url: `/inventory/managedObjects/${mo.id}`
       });
     }
   });
 });
 
-Cypress.Commands.add("addChildDevice", (parentDevice, childDevice) => {
+Cypress.Commands.add('addChildDevice', (parentDevice, childDevice) => {
   cy.apiRequest({
     url: `/inventory/managedObjects?q=$filter=(name eq '${parentDevice}' and has('c8y_IsDevice'))`,
-    method: "GET",
+    method: 'GET'
   }).then((response: any) => {
     const parentDeviceId = response.body.managedObjects[0].id;
     cy.apiRequest({
-      method: "GET",
-      url: `/inventory/managedObjects?q=$filter=(name eq '${childDevice}' and has('c8y_IsDevice'))`,
+      method: 'GET',
+      url: `/inventory/managedObjects?q=$filter=(name eq '${childDevice}' and has('c8y_IsDevice'))`
     }).then((response: any) => {
       const childDeviceId = response.body.managedObjects[0].id;
       cy.log(childDeviceId);
       cy.apiRequest({
         url: `/inventory/managedObjects/${parentDeviceId}/childDevices`,
-        method: "POST",
+        method: 'POST',
         body: {
-          managedObject: { id: `${childDeviceId}` },
-        },
+          managedObject: { id: `${childDeviceId}` }
+        }
       });
     });
   });
 });
 
-Cypress.Commands.add("createEvent", (request) => {
+Cypress.Commands.add('createEvent', request => {
   getDeviceId(request.deviceName).then((response: any) => {
     const deviceId = response.body.managedObjects[0].id;
     const formattedTimestamp = setPastDate(request);
     const body = {
       source: {
-        id: deviceId,
+        id: deviceId
       },
       type: request.type,
       text: request.text,
-      time: formattedTimestamp,
+      time: formattedTimestamp
     };
     cy.apiRequest({
-      method: "POST",
-      url: "/event/events",
+      method: 'POST',
+      url: '/event/events',
       headers: {
-        Accept: "application/vnd.com.nsn.cumulocity.event+json",
+        Accept: 'application/vnd.com.nsn.cumulocity.event+json'
       },
       body: body,
-      failOnStatusCode: false,
+      failOnStatusCode: false
     })
-      .its("status")
-      .should("eq", 201);
+      .its('status')
+      .should('eq', 201);
+  });
+});
+
+Cypress.Commands.add('addAssetToDevice', (parentDevice, asset) => {
```

Comment:
In what scenario does a device have an asset as a child?

Reply:
No, currently we are able to add child assets via the API.